pkg/kv/kvclient/kvcoord/txn_coord_sender.go

```diff
@@ -495,7 +496,7 @@ func (tc *TxnCoordSender) Send(
 		return nil, tc.finalizeNonLockingTxnLocked(ctx, ba)
 	}
 
-	ctx, sp := tc.AnnotateCtxWithSpan(ctx, OpTxnCoordSender)
+	ctx, sp := tracing.ChildSpan(ctx, OpTxnCoordSender)
```

Comment:
Do we care about the extra context stuff that's pulled in from ambient context?

Reply:
That's a good point, there is more stuff in there than I thought. Maybe I should make a change that calls `ChildSpan` instead of `EnsureChildSpan` in `AnnotateCtx`?

Reply:
Trying that out, this commit is now replaced with `log: don't call EnsureChildSpan in AnnotateCtx`.